test/functional/cypress/specs/referrals/referral-help-spec.js

```diff
@@ -0,0 +1,55 @@
+const selectors = require('../../../../selectors')
+const { assertBreadcrumbs } = require('../../support/assertions')
+
+describe('Referral help', () => {
+  context('when viewing referral help', () => {
+    before(() => {
+      cy.visit('/referrals/cc5b9953-894c-44b4-a4ac-d0f6a6f6128f/help')
+    })
+
+    it('should render breadcrumbs', () => {
+      assertBreadcrumbs({
+        Home: '/',
+        Referral: null,
+      })
+    })
+
+    it('should render the heading', () => {
+      cy.get(selectors.localHeader().heading).should(
+        'have.text',
+        'Help to complete this Referral'
+      )
+    })
+
+    it('should render the content and conform to design', () => {
+      cy.get('#referral-help h2:first-child')
+        .should('have.text', 'More information needed')
+        .next()
+        .should('have.prop', 'tagName', 'P')
```

Comment:
There is a simpler way:
```suggestion
        .should('match, 'p')
```